test_embed_functionality.py

```diff
@@ -0,0 +1,35 @@
+from playwright.sync_api import Page, expect
+import os
+
+# Get the current working directory
```

Comment:
This test file should be in the tests directory, which would probably change this section and where it would find the index page